src/wasm-lib/kcl/src/execution/geometry.rs

```diff
@@ -419,6 +429,8 @@ pub struct Sketch {
     /// The original id of the sketch. This stays the same even if the sketch is
     /// is sketched on face etc.
     pub artifact_id: ArtifactId,
+    #[ts(skip)]
+    pub original_id: uuid::Uuid,
```

Comment:
These two fields are technically a duplicate of each other. That's why I renamed it in #5205. I guess it's fine, though, because they're semantically representing two different things.

Reply:
Hmm. Yeah I'm OK leaving it because one might change in the future, but we could also come back and remove it. Whatever you'd prefer.